modules/osdk-scorecard-config.adoc

```diff
@@ -0,0 +1,183 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-scorecard.adoc
+
+[id="osdk-scorecard-config_{context}"]
+= Scorecard configuration
+
+The scorecard tool uses a configuration file that allows you to configure
+internal plug-ins, as well as several global configuration options.
+
+[id="osdk-scorecard-config-file_{context}"]
+== Configuration file
+
+The default location for the scorecard tool's configuration is the
+`<project_dir>/.osdk-scorecard.*`. The following is an example of a
+YAML-formatted configuration file:
+
+.Scorecard configuration file
+[source,yaml]
+----
+scorecard:
+  output: json
+  plugins:
+    - basic: <1>
+        cr-manifest:
+          - "deploy/crds/cache.example.com_v1alpha1_memcached_cr.yaml"
+          - "deploy/crds/cache.example.com_v1alpha1_memcachedrs_cr.yaml"
+    - olm: <2>
+        cr-manifest:
+          - "deploy/crds/cache.example.com_v1alpha1_memcached_cr.yaml"
+          - "deploy/crds/cache.example.com_v1alpha1_memcachedrs_cr.yaml"
+        csv-path: "deploy/olm-catalog/memcached-operator/0.0.3/memcached-operator.v0.0.3.clusterserviceversion.yaml"
+----
```

Comment:
All the tested cr-manifest and csv-path should be really exist.

Reply:
@emmajiafan Can you clarify what you would like to see here? Should it just be clearer that you can set more than 2 CRs, for example?